### DISCUSSION TITLE
GitHub Stars

### DISCUSSION THREAD

**Discussion by fakerybakery** [yl4579/StyleTTS2#77] [Q&A] [answered]
What happened in November?!?

[![Star History Chart](https://api.star-history.com/svg?repos=yl4579/StyleTTS2&type=Date)](https://star-history.com/#yl4579/StyleTTS2&Date)

**Comment by nshmyrev** [comment, marked as answer]
Link was posted on Ycombinator https://news.ycombinator.com/item?id=38335255
